scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -103,7 +103,7 @@ def get_memballoon_type():
     </memballoon>"""
 
     return """    <memballoon model='virtio' autodeflate='on'>
-      <address type='pci' slot='0x05'/>
+      <address type='pci' slot='0x15'/>
```

Comment:
This will collide with the disk slots when we have more than 5 disks (a raid + drbd + ceph volumes might easily sum up to this):
https://github.com/SUSE-Cloud/automation/blob/master/scripts/lib/libvirt/libvirt_setup.py#L269

So maybe we need a more intelligent way to assign the PCI ids than assuming that small ranges suffice.